src/common/com/intellij/plugins/haxe/ide/generation/CreateGetterSetterFix.java

```diff
@@ -137,7 +137,7 @@ private static void buildSetter(StringBuilder result, String name, String typeTe
   }
 
   private static void build(StringBuilder result, String name, String typeText, boolean isGetter) {
-    result.append("public function ");
+    result.append("private function ");
```

Comment:
getter setter need not be public ... they are internal implementation ... private is better default
it is the property declaration that needs to be public
This is the one of the changes Haxe 3.x